docker/docker/src/main/java/org/arquillian/cube/docker/impl/await/PollingAwaitStrategy.java

```diff
@@ -78,14 +86,14 @@ public boolean await() {
         }
 
         Collection<Integer> pingPorts = this.ports;
-        if(ports == null) {
+        if (ports == null) {
             pingPorts = portBindings.getBoundPorts();
         }
         for (Integer port : pingPorts) {
-            switch(this.type) {
+            switch (this.type) {
                 case "ping": {
```

Comment:
Currently it is an "SPI" I mean you can write your custom await strategy following an interface. This started as a simple await method that has been grown from time to time.